src/main/java/org/apache/sling/jcr/resource/internal/helper/jcr/JcrNodeResource.java

```diff
@@ -33,6 +33,7 @@
 import javax.jcr.Property;
 import javax.jcr.RepositoryException;
 import javax.jcr.Value;
+import org.apache.jackrabbit.JcrConstants;
```

Comment:
Please use the constants from https://docs.adobe.com/docs/en/spec/javax.jcr/javadocs/jcr-2.0/constant-values.html instead and remover the static imports for org.apache.jackrabbit.JcrConstants as well.